shared/src/constants/empty-length.constant.ts

```diff
@@ -0,0 +1,3 @@
+const EMPTY_LENGTH = 0;
```

Comment:
```suggestion
const EMPTY_ARRAY_LENGTH = 0;
```
EMPTY_LENGTH looks a little bit weird and confusing

Reply:
I agree with you and have changed the name. Unfortunately, I can't use your suggestion because it's used not only for arrays.